aip/general/0122.md

```diff
@@ -297,6 +292,16 @@ embedded resources.
 - If using the resource name is not possible and using the ID component alone is
 strictly necessary, the field **should** use an `_id` suffix (e.g. `shelf_id`).
 
+The field **should not** be of type `message` using the `message` that
+implements the resource, __except__ for one of following conditions:
+
+- The API is internal-only, has tight lifecycle relationships, and has a
+  permission model that enables inherited access to embedded resources.
+- The embedding of the resource is done as part of the AIP-162 revisions
```

Comment:
How about:

- The embedding of the resource is `OUTPUT_ONLY`, similar to AIP-162 revisions snapshot field.
- The embedded appears in messages used only for responses, similar to AIP-132 List response messages.

?